tests/integration/test_enhanced_cnn_lstm_training.py

```
@@ -0,0 +1,424 @@
+Integration tests for Enhanced CNN+LSTM Training Pipeline
+
+This module tests the complete training workflow including:
+- Dataset building/loading
+- Model initialization
+- Training with monitoring
+- Hyperparameter optimization
+- Model saving/loading
+- Metrics calculation
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)